src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -127,6 +127,19 @@ class IOUAmountPage extends React.Component {
         return amount.replace(/,/g, '');
     }
 
+    /**
+     * Adds a leading zero to amount if user entered just the decimal separator
```

Comment:
Posting same suggestion rajat made since it looks like the code was modified since he made it.
```suggestion
     * Adds a leading zero to the amount if user entered just the decimal separator
```